test/validations.test.js

```diff
@@ -525,6 +525,52 @@ describe('validations', function() {
         })).should.be.false;
       });
     });
+
+    it('should validate uniqueness case insensitive', function(done) {
```

Comment:
Please do not start test names with `should`, even though we have followed some bad practices in our previous code. Please see our [style-guide](https://github.com/strongloop/loopback-contributor-docs/blob/master/style-guide.md#test-naming)

Please fix it for other tests as well.